lab01/index.html

```diff
@@ -0,0 +1,63 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <title>Incremental Game</title>
+        <h1>Incremental Game Framework</h1>
+        <link rel="stylesheet" type="text/css" href="app.css">
+    </head>
+    
+    <body>
+        
+        <div class="lines">
+            Stroy Line 1<br/>
+            Stroy Line 2<br/>
+            Stroy Line 3<br/>
+            Stroy Line 4<br/>
+        </div>
```

Comment:
Suggestion here would be using `<p>` for each line rather than using `<br>` on each line.